abstio/src/io_web.rs

```diff
@@ -20,7 +20,7 @@ use crate::{path_player, Manifest};
 static SYSTEM_DATA: include_dir::Dir = include_dir::include_dir!(
     "../data/system",
     "assets/",
-    "-assets/music/",
+    // "assets/music/",
```

Comment:
Intentional? We don't want to bake the Santa music in the .wasm, because it's large and doesn't even work on the web anyway

Reply:
Maybe I've misunderstood though - what's the purpose of the "-" character? I assumed it was an intentional typo to have the directory not be found. 

I thought commenting out was more clear than the intentional typo approach.

Reply:
oh, I think I was just wrong about what the "-" does. It means exclude, otherwise it gets included recursively because its parent is included. 

heh, i'm trying and failing to find any documentation on this feature, but I'll take a look at the macro source and revert this change.

Reply:
https://github.com/dabreegster/include_dir/tree/union
You're not finding docs, because I hacked in this "exclude some globs" feature to a fork of the repo and haven't upstreamed it yet. :\

But yeah, the `-` is intentional and useful

Reply:
ah. Yeah, I really shouldn't have touched this sorry.